src/rqt_publisher/publisher.py

```diff
@@ -31,18 +31,17 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 from __future__ import division
+import importlib
 import math
 import random
 import time
 
 from python_qt_binding.QtCore import Slot, QSignalMapper, QTimer, qWarning
 
-import roslib
-import rospy
-import genpy
+import rclpy
 from rqt_gui_py.plugin import Plugin
 from .publisher_widget import PublisherWidget
-from rqt_py_common.topic_helpers import get_field_type
+from rqt_py_common.topic_helpers import get_message_class, get_property_type
```

Comment:
get_message_class is getting moved to `rqt_py_common.message_helpers`

Reply:
Fixed